core/contractsdk/cpp/src/xchain/trust_operators/trust_operators.cc

```diff
@@ -0,0 +1,41 @@
+#include "xchain/trust_operators/tf.pb.h"
+#include "xchain/trust_operators/trust_operators.h"
+
+extern "C" uint32_t xvm_tfcall(const char* inputptr, size_t inputlen,
+                         char** outputptr, size_t* outputlen);
+
+bool tfcall(const TrustFunctionCallRequest& request, TrustFunctionCallResponse* response) {
+    std::string req;
+    char* out = nullptr;
+    size_t out_size = 0;
+    request.SerializeToString(&req);
+    auto ok = xvm_tfcall(req.data(), req.size(), &out, &out_size);
+    // 注意是返回1表示失败
+    if (ok) {
+        return false;
+    }
+    auto ret = response->ParseFromString(std::string(out, out_size));
+    free(out);
+    return ret;
+}
+
+TrustOperators::TrustOperators(const std::string& addr):_address(addr){}
+
+bool TrustOperators::store(const uint32_t svn, const std::string& args,
```

Comment:
It is recommended to combine `put_object` to directly store data here, which is more in line with the semantics of `store`

Reply:
done.